LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 PyScript
```

Comment:
Let me know if someone else has the Copyright

Reply:
Had a sec and updated it but feel free to also close it.

Reply:
Thanks for the prompt. This is obviously a very new project.

I really appreciate the help getting it onto conda-forge!

Reply:
Yes of course totally understood. Just wanted to get this into conda-forge so that I can try it :)